torchvision/models/segmentation/deeplabv3.py

```diff
@@ -47,21 +48,47 @@ def __init__(self, in_channels, out_channels, dilation):
         super(ASPPConv, self).__init__(*modules)
 
 
-class ASPPPooling(nn.Sequential):
+class ASPPPooling(nn.Module):
+    __constants__ = ['mods']
+    _version = 2
+
     def __init__(self, in_channels, out_channels):
-        super(ASPPPooling, self).__init__(
+        super(ASPPPooling, self).__init__()
+        self.mods = nn.Sequential(
```

Comment:
So, I was thinking, in this particular case we do not use any of the features special from `nn.Sequential` that have custom implementation in the JIT, like desugaring iterator over the modules.

In this case, what about unrolling the forward of the module, something like
```python
class ASPPPooling(nn.Module):
    ...
    def __init__(self, ...):
        for i, module in enumerate([
                nn.AdaptiveAvgPool2d(1), ...
            ]
            self.add_module(str(i), module)

    def forward(self, x):
        ...
        x = self['0'](x)
        x = self['1'](x)
        ...
        return ...
```
This way, we don't need to make a breaking change in the model.
Thoughts?

Reply:
Won't this still break backwards compat ?

Reply:
No, because the state-dicts will be the same (as we will be using the same names as we had before).

We only serialize the `state_dict` (which is a dict of tensors), so we are free to change how it is implemented, as long as the computations are the same

Reply:
Unfortunately you can't index into a module currently - `self['1']`

Reply:
Ok then, but that's unfortunate :-/